test/fixtures.yml

```diff
@@ -0,0 +1,44 @@
+usps:
```

Comment:
My apologies. This code has been running in production for the better part of a year. I only recently rebased, and this got added back in. Will fix.